cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -30,7 +30,11 @@ bundle agent cfe_internal_setup_knowledge
         or => {
                 "enterprise_3_7_3", "enterprise_3_7_4", "enterprise_3_7_5",
                 "enterprise_3_10_0"
-              };
+        };
+
+  vars:
+      "install_logs" -> {"ENT-4564"}
+        slist => findfiles("/var/log/CFEngine*Install.log");
```

Comment:
Alternatively, instead of having an intermediary variable, you could use a regex in the promiser.

Reply:
I don't know, I like that it explicitly acts on files that exist. And globs are easier to read and more come in combination with file paths. YMMV.

Reply:
Indeed, that's the advantage of the intermediary variable. You are documenting a specific intent.
Cool, let's keep it the way it is. However, this should be a different path on windows.

Reply:
>  Cool, let's keep it the way it is. However, this should be a different path on windows.

AFAICT, I'm not changing anything here regarding windows. Also, I don't think we have such an installation log on windows in the first place. So do you think we should just move this promise under a `!windows::` guard?